docs/upgrade/2.6.1_to_2.7.0.rst

```diff
@@ -0,0 +1,108 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.6.1 to 2.7.0
+===========================
+
+.. note::
+
+   This release will remove support for submission PGP keys with legacy 
+   SHA-1-based binding signatures. The SecureDrop Journalist Interface
+   will not start when the instance has been configured with such a key.
+   If you have set up SecureDrop according to our documentation, you are
+   not using such keys; no SecureDrop instances known to us are affected
+   by this change.
+
+   If you are unsure if you will be affected by this change, you can 
+   reach out to us for support. Our recommended course of action is to
+   check your Submission Public Key, available at the /public-key
+   endpoint of your SecureDrop Source Interface onion url, using the
+   ``sq-keyring-linter`` program installable on your Admin Workstation.
```

Comment:
Is there a way we can mention Tails 5.19 here or something instead of the awkward "installable" (which I think was my wording, because it was before the Tails release)?

Reply:
Definitely, please let me know what you think about how I incorporated Tails 5.19 here in the latest commit (happy to re-work it if need be). Thanks!